conda/models/records.py

```diff
@@ -244,6 +244,11 @@ class PathsData(Entity):
     paths = ListField(PathData)
 
 
+class Metadata(set):
+    def __str__(self) -> str:
+        return " ".join(self)
```

Comment:
I never metadata I didn't like

Is this better than set's builtin __str__?

Reply:
see https://github.com/conda/conda/pull/13053/commits/ce45724620ab7f09605e909e271b8e2ec1294242